src/components/ui/coaching-sessions/coaching-session-title.tsx

```diff
@@ -43,14 +43,14 @@ const CoachingSessionTitle: React.FC<{
 
   if (isLoading) {
     return (
-      <h4 className="font-semibold break-words w-full px-2 md:px-4 lg:px-6 md:text-clip">
```

Comment:
This works, I had originally preferred the padding, but I realized in seeing this that my eye just needs to get used to the straight alignment. Thanks for making this change.